configs/browser_config.json

```diff
@@ -34,7 +34,8 @@
     },
     "noProxy": "localhost,127.0.0.1",
     "proxyType": "manual",
-    "binary": "C:\\Users\\pathTo\\electronApp.exe"
+    "binary": "/Applications/JioMeet-Lite.app/Contents/MacOS/JioMeet-Lite",
```

Comment:
remove this value. If there is a test app, let's add it here - https://github.com/anandbagmar/sampleAppsForNativeMobileAutomation/ - and give that url